drive/snippets/drive_v3/file_snippets/export_pdf.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// [START drive_export_pdf]
+
+/**
+ * Download a Document file in PDF format
+ * @param{string} realFileId file ID
+ * @return{obj} file status
+ * */
+async function exportPdf(realFileId) {
+  // Get credentials and build service
```

Comment:
Move closer to statements so it reads better. Feels a little disconnected with the require statements in between